packages/components/psammead-visually-hidden-text/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 | Version | Description |
 | --------------------- |
-| 0.3.10  | [PR#323](https://github.com/bbc/psammead/pull/323) Update storybook badge url |
```

Comment:
This was a [version typo](https://github.com/bbc/psammead/pull/323/files#diff-5d2b9269702cadef473c4286f20d06ccR5)